layouts/partials/head.html

```diff
@@ -1,32 +1,47 @@
+<!-- Variables
+     =========
+  The root layout should include the "common_variables.html" partial before
+  anything else. That file defines useful variables and adds them to the global
+  .Scratch dictionary, which can then be used to pull only the variables we need
+  into any given partial at any given scoping level.
+
+{{ $project              := .Scratch.Get "project"              }}
+{{ $version              := .Scratch.Get "version"              }}
+{{ $site_relative_url    := .Scratch.Get "site_relative_url"    }}
+{{ $version_path         := .Scratch.Get "version_path"         }}
+{{ $project_relative_url := .Scratch.Get "project_relative_url" }}
+{{ $canonical_link       := .Scratch.Get "canonical_link"       }}
+-->
 <head class="tutorials">
   <meta charset="utf-8">
   <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
   <meta name="viewport" content="width=device-width">
 
-  <meta name="project" content="{{ .Params.project }}">
-  <meta name="version" content="{{ .Params.project_version }}">
-
-  <!-- TODO: Adding these leading slashes is... pretty dumb. IMO. We should
-             really clean this up. -->
-  <meta name="site_relative_url" content='/{{ .Scratch.Get "site_relative_url" }}'>      <!-- ex; /riak/kv/2.1.4/installing/ -->
-  <meta name="version_url" content='/{{ .Scratch.Get "version_url" }}'>                  <!-- ex; /riak/kv/2.1.4 -->
-  <meta name="project_relative_url" content='{{ .Scratch.Get "project_relative_url" }}'> <!-- ex; installing/ -->
-
-{{ if .Params.version_history.in }}
-  <meta name="version_history_in" content="{{ .Params.version_history.in }}">
-{{ end }}
-{{ if .Params.version_history.locations }}
-  <meta name="version_history_locations" content="{{ .Params.version_history.locations | jsonify }}">
-{{ end }}
+  <title>{{ .Title }}</title>
```

Comment:
Good catch. Wound up going with the below (w/o the ":") b/c some (all?) of our supertext elements explicitly include a trailing ":", and I don't want duplicates.

```
  <title>{{ if .Params.title_supertext }}{{ .Params.title_supertext }} {{ end }}{{ .Title }}</title>
```